### PR TITLE
fix: add Anurag to the list

### PR DESCRIPTION
**Pull request by kr-anurag**
added my profile here :yeye:

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/722"></a>

### CODE REVIEW

File: public/data/anurag.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://twitter.com/kr_anurag_",
+        "icon": "twitter"
+      },
+      {
+          "name": "Join my Discord community",
+          "url": "https://discord.gg/MqsS6axMrZ",
+          "icon": "discord"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/anurag.json:22]: Missing an endline.
```suggestion
}

```

File: public/data/anurag.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Anurag",
+    "bio": "Teen | Front-end Developer | Student",
+    "avatar": "https://github.com/kr-anurag.png",
+    "links": [
+      {
+        "name": "Checkout my Projects on Github",
```
Comment (Panquesito7) [inline public/data/anurag.json:7]:
```suggestion
        "name": "Check out my Projects on GitHub",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by kr-anurag** [comment]
done!

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**